.github/ISSUE_TEMPLATE/10-bug-report.yml

```diff
@@ -0,0 +1,81 @@
+name: Bug report
+description: Use this template for reporting a bug.
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: "# syslog-ng"
+  - type: textarea
+    attributes:
+      label: Version of syslog-ng
+      description: |
+        Output of:
+        ```
+        $ syslog-ng --version
+        ```
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Platform
+      description: Name and version of OS
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Debug bundle
```

Comment:
We rarely used this to resolve bug reports yet.

I would move this to the bottom, or just leave out. We can always ask for a debun bundle.